src/Uno.UI/UI/Xaml/Controls/CommandBar/CommandBar.cs

```diff
@@ -8,6 +8,8 @@
 using Windows.UI.Xaml.Markup;
 using Windows.UI.Xaml.Controls.Primitives;
 using Uno.Logging;
+using UIKit;
```

Comment:
This won't compile on other platforms. You'll need to add conditionals. (FindFirstChild is in the Uno.UI namespace on Android, I think on other platforms it's a normal method on FrameworkElement.)

Reply:
Actually the NativeCommandBarPresenter doesn't seems to exist on other platform. I just set it for Android and iOS. We can see that point tomorrow if needed